read_yields.py

```diff
@@ -420,11 +665,11 @@ def get(self,M=0,Z=-1,quantity='',specie=''):
         else:
             data=self.yield_data[idx]
             if specie=='':
-                idx_col=self.col_attrs.index(quantity)
+                idx_col=self.data_cols.index(quantity)
                 set1=data[idx_col]
                 return set1
             else:
-                idx_col=self.col_attrs.index('Yields')
+                idx_col=self.data_cols.index('Yields')
                 set1=data[idx_col]
                 specie_all= data[0]
                 for k in range(len(set1)):
```

Comment:
I have to say that I don't know read_yields.py that much.. I will trust you on this.